.github/build-ci/manifests/cice5/access-om2~deterministic-intel.spack.yaml.j2

```diff
@@ -0,0 +1,11 @@
+spack:
+  specs:
+    - 'access-om2@=stable ~deterministic %{{ intel_compiler }} ^cice5 build_system=cmake nxglob=360 nyglob=300 blckx=15 blcky=300 mxblcks=1'
+    - 'access-om2@=stable ~deterministic %{{ intel_compiler }} ^cice5 build_system=makefile'
```

Comment:
Any particular reason for using `@=` instead of just `@`? If not, just stick to `@`.